README.md

```diff
@@ -101,26 +102,28 @@ await insert(db, {
   description: 'Experience immersive sound quality with these noise-cancelling wireless headphones.',
   price: 99.99,
   meta: {
-    rating: 4.5
-  }
+    rating: 4.5,
+  },
 })
 
 await insert(db, {
   name: 'Smart LED Bulb',
-  description: 'Control the lighting in your home with this energy-efficient smart LED bulb, compatible with most smart home systems.',
+  description:
+    'Control the lighting in your home with this energy-efficient smart LED bulb, compatible with most smart home systems.',
```

Comment:
Pay attention to linting and prettier actions: here is putting the string on a new line (not needed). Do you happen to have a global prettier/eslint config?

Reply:
I think it was VS Code autolinting. Removed it!